opsmngr/automation_config.go

```diff
@@ -219,6 +219,7 @@ type Member struct {
 	SecondaryDelaySecs *float64                `json:"secondaryDelaySecs,omitempty"` // SecondaryDelaySecs replaces SlaveDelay since 5.0+
 	Tags               *map[string]interface{} `json:"tags,omitempty"`
 	Votes              float64                 `json:"votes"`
+	Horizons           *map[string]string      `json:"horizons,omitempty"`
```

Comment:
can we add a godoc about this being readonly?
```suggestion
	Horizons           *map[string]string      `json:"horizons,omitempty"` // Horizons is managed by the operators and should not be manually edited 
```

or something like it?

Reply:
will do!